src/constants/ProgressBar.js

```diff
@@ -0,0 +1,12 @@
+const ProgressBar = Object.freeze({
+  Sizes: {
+    SMALL: 'small',
+    LARGE: 'large',
+  },
+  Variants: {
+    PRIMARY: 'primary',
+    SECONDARY: 'secondary',
+  },
+});
+
+export default ProgressBar;
```

Comment:
Piggy backing on the suggestion above, this would be rewritten this way: 

```suggestion
export const Sizes = Object.freeze({
    SMALL: 'small',
    LARGE: 'large',
});

export const Variants = Object.freeze({
    PRIMARY: 'primary',
    SECONDARY: 'secondary',
});
```